lib/node_modules/@stdlib/blas/base/caxpy/test/test.caxpy.native.js

```diff
@@ -80,14 +80,14 @@ tape( 'the function has an arity of 6', opts, function test( t ) {
 	t.end();
 });
 
-tape( 'the function scales elements from `cx` by `ca` and adds the result to `cy`', opts, function test( t ) {
+tape( 'the function scales elements from `x` by `ca` and adds the result to `y`', opts, function test( t ) {
 	var expected;
 	var viewY;
 	var ca;
```

Comment:
In the other files, you changed `ca` to `alpha`. Do you want to do the same here and elsewhere in this file?